api/services/References.js

```diff
@@ -8,7 +8,8 @@ exports.approve = function (ref, approve) {
       or: [
         {type: 'casual'},
         {type: 'shiny'},
-        {type: 'event'}
+        {type: 'event'},
+        {type: 'bank'}
```

Comment:
The backend still calls it "bank". We can fix this later (e.g. next update), but fixing it now is probably not a good idea since it will lead to bugs, and this particular fix isn't going through a beta-testing cycle.